shell/browser/api/atom_api_web_request_ns.cc

```diff
@@ -176,7 +207,7 @@ void ReadFromResponse(v8::Isolate* isolate,
   if (!response->Get("statusLine", &status_line))
     status_line = headers.second;
   v8::Local<v8::Value> value;
-  if (response->Get("responseHeaders", &value)) {
+  if (response->Get("responseHeaders", &value) && value->IsObject()) {
```

Comment:
wouldn't you want to flip this condition? so that if the value isn't an object it's immediately false and there is no Get called on that response*?
```suggestion
  if (value->IsObject() && response->Get("responseHeaders", &value)) {
```

Reply:
@allydurks the type of value is undefined until its extracted from the `response` dictionary, so the condition can't be flipped.